css/components/navigation.css

```diff
@@ -0,0 +1,57 @@
+/* === css for navigation ==== */
+
+.navigation-container{
+    display: flex;
+    width: 100%;
+    align-items: center;
+    padding: 1.5rem .7rem;
+    justify-content: space-between;
+    background-color: #333;
+}
+.left h5{
+    font-size: var(--normal-font-size);
+    color: #fff;
+}
+.middle{
+    position: relative;
+    display: flex;
+    align-items: center;
+}
+.middle input{
+    padding: .5rem;
+    width: 15rem;
+    outline: none;
```

Comment:
try not to use outline: none as it creates issues for people navigating the web with a keyboard.

Reply:
sure , i just removed that.